server/executor/trigger/grpc.go

```diff
@@ -144,6 +145,52 @@ func (t *grpcTriggerer) Resolve(ctx context.Context, test model.Test, opts *Trig
 	return test, nil
 }
 
+func (t *grpcTriggerer) Variables(ctx context.Context, test model.Test, executor expression.Executor) (expression.VariablesMap, error) {
+	triggerVariables := expression.VariablesMap{}
+	grpc := test.ServiceUnderTest.GRPC
+
+	if grpc == nil {
+		return triggerVariables, fmt.Errorf("no settings provided for GRPC triggerer")
+	}
+	address, err := executor.StatementTermsByType(WrapInQuotes(grpc.Address, "\""), expression.EnvironmentType)
+	if err != nil {
+		return triggerVariables, err
+	}
+	triggerVariables = triggerVariables.MergeStringArray(address)
```

Comment:
Small / not important suggestion: to make it easier to read it makes sense to group the guard clauses if the statements that we are checking, like the `grpc := ...` line and `if grpc == nil { ...` line

```suggestion
	triggerVariables := expression.VariablesMap{}
	
	grpc := test.ServiceUnderTest.GRPC
	if grpc == nil {
		return triggerVariables, fmt.Errorf("no settings provided for GRPC triggerer")
	}
	
	address, err := executor.StatementTermsByType(WrapInQuotes(grpc.Address, "\""), expression.EnvironmentType)
	if err != nil {
		return triggerVariables, err
	}
	
	triggerVariables = triggerVariables.MergeStringArray(address)
```

Reply:
Done 😄